stoobly_agent/config/data_dir.py

```diff
@@ -31,13 +39,19 @@ def instance(cls):
 
         return cls._instance
 
+    @classmethod
+    def reset(cls) -> None:
```

Comment:
If this is for testing, see https://github.com/Stoobly/stoobly-agent/blob/cba786c9e0e3bc6d13f7f1cf1634dc8c1d45d215/stoobly_agent/test/test_helper.py#L16